tests/test_basic_ops_dict.py

```diff
@@ -66,28 +66,69 @@ def test_getattr_dict() -> None:
     assert {"b": 1} == c.a
 
 
-def test_mandatory_value() -> None:
-    c = OmegaConf.create({"a": "???"})
-    with pytest.raises(MissingMandatoryValue, match="a"):
-        c.a
-
-
-def test_nested_dict_mandatory_value() -> None:
-    c = OmegaConf.create(dict(a=dict(b="???")))
-    with pytest.raises(MissingMandatoryValue):
-        c.a.b
-
-
-def test_subscript_get() -> None:
-    c = OmegaConf.create("a: b")
-    assert isinstance(c, DictConfig)
-    assert "b" == c["a"]
-
-
-def test_subscript_set() -> None:
-    c = OmegaConf.create()
-    c["a"] = "b"
-    assert {"a": "b"} == c
+@pytest.mark.parametrize(
+    "key",
+    ["a", 1],
+)
+class TestDictKeyTypes:
+    def test_mandatory_value(self, key: Any) -> None:
```

Comment:
Shouldn't all `key: Any` actually be `key: DictKeyType`?

Reply:
Oh yes, absolutely!

Reply:
Just a little import sorting issue left to fix ;)